### ISSUE TITLE
Add a --accounts-file parameter

### ISSUE THREAD

**Issue by riahc3** [farshadz1997/Microsoft-Rewards-bot#229]
Hello

Please add a --accounts-file parameter ; This way every 24 hours, 6 accounts can be run in a parallel script that calls your script.

Thank you

**Comment by Prabh-Saini** [comment]
I think `--accounts-list` sounds a little nicer. However, thank you for the request. This is a wonderful feature that will be implemented as soon as possible. 🚀

**Comment by riahc3** [comment]
Yeah sure,

--accounts-list accounts1.json
--accounts-list accounts1.json,accounts2.json,

**Comment by farshadz1997** [comment]
that's good idea

**Comment by riahc3** [comment]
> that's good idea

Thank you

**Comment by richard892** [comment]
I am non coder, how to run 5 account in parallel?
thanks in advance

**Comment by thefakeknightstellar** [comment]
> I am non coder, how to run 5 account in parallel? thanks in advance

I don't think this is related to this issue but just follow the instructions in https://github.com/farshadz1997/Microsoft-Rewards-bot/blob/master/setup.md u dont need to be a coder to use this
